modules/packages/SharedObject.chpl

```diff
@@ -117,7 +114,8 @@ module SharedObject {
 
        :arg p: the class instance to manage. Must be of class type.
      */
-    proc Shared(p, type t=p.type) {
+    proc init(p, type t=p.type) {
```

Comment:
@mppf: Belatedly, I'm thinking that the `type t` argument was a requirement in the constructors
world that should be moved from the argument list to the initializer body in the current world.
I.e., a user is not expected to be able to pass a type 't' into the initializer, is that right?  (and
ditto for other ones below).

Reply:
(i.e., I don't think I was thinking hard enough about the interface when mechanically converting these).

Reply:
Yes that's right. It was there simply to satisfy the compiler.